doc/code/qml_drawer.rst

```diff
@@ -50,14 +57,3 @@ Currently Available Styles
 +-----+-----+-----+
 +|bws|+|bwd|+|def|+
 +-----+-----+-----+
-
-
-Developer Methods
------------------
-
-.. automodapi:: pennylane.drawer
```

Comment:
How come this was changed?

Reply:
`draw` and `draw_mpl` are not developer methods. In fact, they're the most important methods of the module, so I thought it would be important to move them to the top.

I think `tape_text` and `tape_mpl` are on the boundary between user-facing and developer-only.  They have quite a bit of use for anyone working primarily with tapes instead of just qnodes.